ports/tree-sitter-c/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "tree-sitter-c",
+  "version": "0.23.5",
+  "description": "C grammar for tree-sitter",
+  "homepage": "https://github.com/tree-sitter/tree-sitter-c",
+  "license": "MIT",
+  "dependencies": [
+    {
```

```suggestion
    "tree-sitter",
    {
```

vcpkg policy requires that you declare a dependency on the components needed for the port to be usable. For example, header-only libraries don't technically need to link dependencies during build, but we still want users to have the dependencies to avoid usage issues.